cmd/clidoc/main.go

```diff
@@ -173,13 +173,7 @@ func writeMessages(path string) error {
 	})
 
 	var w bytes.Buffer
-	for _, m := range toSort {
-		w.WriteString(fmt.Sprintf(`###### %s (%d)
-
-%s
-
-`, m.Text, m.ID, "```json\n"+codeEncode(m)+"\n```"))
-	}
+	w.WriteString("\n\n```json\n" + codeEncode(toSort) + "\n```")
```

Comment:
This will break the existing behavior. Can you please create a new file for this?

Reply:
Where do you want me to put it?

Reply:
Same location, filename `messages.json`

Reply:
Updated!